pom.xml

```diff
@@ -25,10 +25,6 @@ limitations under the License.
     <version>6.1.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
-    <prerequisites>
```

Comment:
This prerequisites tag is only to be used with Maven Plugin project POMs, not plain old modules like Roller's.